packages/react-web/src/inbox/helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { ProfileId, profileId as brandProfileId } from '@lens-protocol/react';
+import { invariant } from '@lens-protocol/shared-kernel';
+import { Conversation } from '@xmtp/react-sdk';
+
+const CONVERSATION_ID_PREFIX = 'lens.dev/dm';
```

Comment:
An open question more here, wondering if all projects would want to use the same conversation id 🤔

Reply:
this is coming from our own docs https://docs.lens.xyz/docs/encrypted-dms and seems the standard for all lens apps integrating xmtp. but as a builder you can also display all xmtp conversations wallet-to-wallet (where with prefix they are considered lens profile-to-profile ones). That is what xmtp calls a "interoperable inbox". https://xmtp.org/docs/concepts/interoperable-inbox

Reply:
got it, make sense